### ISSUE TITLE
Update the update-component-usage script to include USWDS search

### ISSUE THREAD

**Issue by Andrew565** [department-of-veterans-affairs/vets-design-system-documentation#2278]
## Description
As a DST developer, I would like the update-component-usage script to include the results from running the `yarn search uswds` command. I also want it to no longer output the react component usage .csv file, as it is no longer needed.

## Tasks
- [ ] Remove the react component usage .csv file output
- [ ] Add a .txt output with the results from the `yarn search uswds` command

## Acceptance Criteria
- [ ] Above tasks are completed

**Comment by Andrew565** [comment]
Closing as this will no longer be needed once work is done on #2413.
